src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -662,7 +667,10 @@ private SnapshotSummaryModel finishCreateSnapshot(
 
     DataRepoResponse<SnapshotSummaryModel> snapshotResponse =
         dataRepoClient.waitForResponse(user, jobResponse, new TypeReference<>() {});
-    return validateResponse(snapshotResponse, "snapshot create", HttpStatus.CREATED, jobResponse);
+    SnapshotSummaryModel snapshotSummary =
+        validateResponse(snapshotResponse, "snapshot create", HttpStatus.CREATED, jobResponse);
+    logger.info("Snapshot created: {}", snapshotSummary);
```

Comment:
Example log:

```
    2023-02-14 15:47:05,726 INFO  [Test worker] b.terra.integration.DataRepoFixtures: Snapshot created: class SnapshotSummaryModel {
        id: ca4df033-b829-4979-ba3d-c8aa359eb26c
        name: row_id_test60ffac71_4e18_4c0a_b518_784ae6d66bf0
        description: snapshot by row id test
        createdDate: 2023-02-14T20:46:18.309774Z
        profileId: 73569864-728b-4ed8-8a4a-74f66b37d46e
        storage: [class StorageResourceModel {
            region: centralus
            cloudResource: application_deployment
            cloudPlatform: azure
        }, class StorageResourceModel {
            region: centralus
            cloudResource: storage_account
            cloudPlatform: azure
        }, class StorageResourceModel {
            region: centralus
            cloudResource: synapse_workspace
            cloudPlatform: azure
        }]
        secureMonitoringEnabled: false
        consentCode: null
        phsId: phs100321
        cloudPlatform: azure
        dataProject: null
        storageAccount: null
        selfHosted: false
        globalFileIds: false
    }
```

Reply:
Although @nmalfroy, I remember you saying that you had hoped to see the storage account logged in a case like this and it's null.  Any ideas? (I wouldn't hold the merge to fix this, if it came down to it.)